contracts/programs/bubblegum/src/state/authority.rs

```diff
@@ -0,0 +1,88 @@
+use crate::error::BubblegumError;
+use anchor_lang::prelude::*;
+
```

Comment:
I think there should probably be some way of tracking the number of remaining spots in the tree to make sure that we don't grant append authority carelessly. In practice, append auth is probably going to just be the tree creator (very centralized), but there should be a system to allow for shared trees